css-layout-classes/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <title>CSS Layout Classes</title>
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <header>
+          <h1>Choose your character!</h1>
+        </header>
+      </div>
+    </div>
+    <div class="column-full">
+      <article>
+        <div class="row">
+          <div class="column-one-third">
+            <img src="./images/link.png">
+          </div>
+          <div class="column-one-third">
+            <h3>Super Smash Bros. Ultimate</h3>
+            <p>Super Smash Bros. Ultimate received universal acclaim from both critics and players, with some critics calling it the best installment in the series.
+              It received praise for its large amount of content and fine-tuning of existing Smash gameplay elements, although its online mode was widely criticized.
+            </p>
+          </div>
+          <div class="column-one-third">
+            <img src="./images/yoshi.png">
+          </div>
+        </div>
+      </article>
+    </div>
+    <div class="column-full">
+      <article>
+        <div class="row">
+          <div class="column-one-fourth">
+            <img src="./images/kirby.png">
+          </div>
+          <div class="column-one-fourth">
+            <img src="./images/fox.png">
+          </div>
+          <div class="column-one-fourth">
+            <img src="./images/pikachu.png">
+          </div>
+          <div class="column-one-fourth">
+            <img src="./images/ness.png">
+          </div>
+        </div>
+      </article>
+    </div>
+    <div class="row">
+      <div class="column-two-thirds">
+        <article>
+          <h3>Donkey Kong</h3>
+          <p>
+            Donkey Kong is a serires of video games featuring the adventures of a gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1981.
+            The franchise consists mainly of two game genres, but also includes additional spin-off titles of various genres.
+          </p>
+          <p>
+            The games of first genre are mostly single-screen platform/action puzzle types, featuring Donkey Kong as the opponent in an industrial construction setting.
+          </p>
+        </article>
+      </div>
+      <div class="column-one-third">
+        <img src="./images/dk.png">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <img src="./images/mario.png">
+      </div>
+      <div class="column-two-thirds">
+        <article>
```

Comment:
same as above, this element is not needed, but it is not hurting anything either!